### DISCUSSION TITLE
FeatureRequest: Web Login page, when the password is set for the web interface

### DISCUSSION THREAD

**Discussion by Homie13** [arendst/Tasmota#11168] [Ideas]
Hi Theo,
hi folks,

thank you very much for your great project. I like it very much.
I have a Feature Request, it would be very nice, if you could integrate a web Login page,
when the password is set for the web interface. See the photo for an example.

Thanks in advance.

![Web-Tasmota](https://user-images.githubusercontent.com/32713567/109488685-29c64c80-7a86-11eb-9804-e89d07a5e030.png)

**Comment by sfromis** [comment]
Having set `WebPassword`, the browser asking for userid/password serves as a login page.

**Comment by Homie13** [comment]
Yes i know. But i find it would be nicer to have a seperate web page it shows more professional.
